docs/introduction.rst

```diff
@@ -0,0 +1,32 @@
+Introduction
+============
+
+async-service
+~~~~~~~~~~~~~
+
+*Lifecycle management for async applications*
+
+Goals
+-----
+
+This library provides strong lifecycle management for asynchronous applications.
+
+Features:
+
+* Provide a well defined service lifecycle
+* Allow orchestration of services via:
+
+    - Tasks
+    - Daemon Tasks
+    - Child Services
+
+* Well-behaved cancellation of services
```

Comment:
I pulled these out of my ass, I'm sure we can come up with a nicer introduction page :)